lib/rouge/lexers/make.rb

```diff
@@ -11,16 +11,6 @@ class Make < RegexLexer
       filenames '*.make', 'Makefile', 'makefile', 'Makefile.*', 'GNUmakefile'
       mimetypes 'text/x-makefile'
 
-      bsd_special = %w(
-        include undef error warning if else elif endif for endfor
-      )
```

Comment:
These should remain in the lexer but need rules to use them (the fact they don't have rules is the cause of #913).

I know it will mean the warnings are output but can you put this one back? I'll try to see if designing a rule is something I can do quickly and will update if I think of something.